frontend/src/bundles/auth/store/slice.ts

```diff
@@ -1,20 +1,21 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 import { DataStatus } from '~/bundles/common/enums/enums.js';
-import { type ValueOf } from '~/bundles/common/types/types.js';
+import {
+    type AuthResponseDto,
+    type ValueOf,
+} from '~/bundles/common/types/types.js';
 
 import { signIn, signUp } from './actions.js';
 
-type User = object;
-
 type State = {
     dataStatus: ValueOf<typeof DataStatus>;
-    user: User;
+    user: AuthResponseDto;
```

Comment:
```suggestion
    user: UserAuthResponseDto;
```

Reply:
Maybe I should keep the `user: AuthResponseDto;` because when I make it `UserAuthResponseDto` it make conflicts on signIn because it was AuthResponseDto type. It expects the payload to be of type `WritableDraft<UserAuthResponseDto>`, but the actual payload type is `AuthResponseDto`. 
![auth](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/72942456/f5f6849e-f9f6-451b-b8b6-6974c1b0b05c)

Reply:
It's likely that the type was changed and now represents whole response instead of user entity and UserAuthReponseDto was added to represent the user entity. As a result, the original type was replaced, but this change hasn't been noticed here. UserAuthResponseDto should be used since we don't need to store entire response, only info about user.

Reply:
Made as you said)Hope it is fine now